CHANGELOG.md

```
@@ -1,5 +1,18 @@
 # Changelog
 
+## [0.18.0](https://github.com/rivenmedia/riven-frontend/compare/v0.17.0...v0.18.0) (2024-11-06)
+
+
```

Markdownlint

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)